bin/publish

```diff
@@ -1,54 +1,103 @@
 #!/bin/bash -e
 
+# The following is used to:
+# Publish images on pre-release and tag as edge
+# Promote pre-releases to releases and tag as latest
+
 . bin/build_utils
 
-REDHAT_IMAGE='scan.connect.redhat.com/ospid-1c46a2de-1d88-40e6-a433-7114ad0099cb/conjur-openshift-authenticator-client'
+function print_help() {
+  echo "Release Usage: $0"
+  echo "Promote Usage: $0 --source <VERSION> --target <VERSION>"
+  echo " --source <VERSION>: specify version number of local image"
+  echo " --target <VERSION>: specify version number of remote image"
+}
 
-readonly REGISTRY="cyberark"
+while [[ $# -gt 0 ]]; do
+  case "$1" in
+  --source)
+    SOURCE_TAG=$2
```

Comment:
Nit: Should be quotes around `$2` here and on line 23 below. (I know we haven't been terribly consistent on this. :/ )
(Ref: https://google.github.io/styleguide/shellguide.html#s5.7-quoting)

Reply:
Fixed, thanks.